ubuntu_image/builder.py

```diff
@@ -174,6 +182,12 @@ def populate_rootfs_contents(self):
             shutil.copy(self.cloud_init, userdata_file)
         # This is just a mount point.
         os.makedirs(os.path.join(dst, 'boot'))
+        self._next.append(self.populate_rootfs_contents_hooks)
+
+    def populate_rootfs_contents_hooks(self):
+        # This has been moved to a separate step to ease testing.
```

Comment:
I'm not sure that comment adds value.

How about adding something like the following docstring instead?

"""
This step runs the hook(s) in the passed hooks directory.
"""